docs/spec/qualifiers.rst

```diff
@@ -213,41 +213,52 @@ following should be allowed::
 Syntax
 ^^^^^^
 
-``Annotated`` is parameterized with a type and an arbitrary list of
-Python values that represent the annotations. Here are the specific
-details of the syntax:
+``Annotated`` is parameterized with a *base expression* and at least one
+Python value representing associated *metadata*::
 
-* The first argument to ``Annotated`` must be a valid type
+    from typing import Annotated
 
-* Multiple type annotations are supported (``Annotated`` supports variadic
+    Annotated[BaseExpr, Metadata1, Metadata2, ...]
+
+Here are the specific details of the syntax:
+
+* The base expression (the first argument to ``Annotated``) must be valid
+  in the context where it is being used:
+
+    * If ``Annotated`` is used in a place where arbitrary
+      :term:`annotation expressions <annotation expression>` are allowed,
+      the base expression may be an annotation expression.
+    * Otherwise, the base expression must be a valid :term:`type expression`.
+
+* Multiple metadata elements are supported (``Annotated`` supports variadic
   arguments)::
 
     Annotated[int, ValueRange(3, 10), ctype("char")]
 
-* ``Annotated`` must be called with at least two arguments (
-  ``Annotated[int]`` is not valid)
+* There must be at least one metadata element (``Annotated[int]`` is not valid)
 
-* The order of the annotations is preserved and matters for equality
+* The order of the metadata is preserved and matters for equality
   checks::
 
     Annotated[int, ValueRange(3, 10), ctype("char")] != Annotated[
         int, ctype("char"), ValueRange(3, 10)
     ]
 
 * Nested ``Annotated`` types are flattened, with metadata ordered
-  starting with the innermost annotation::
+  starting with the innermost ``Annotated`` expression::
 
     Annotated[Annotated[int, ValueRange(3, 10)], ctype("char")] == Annotated[
         int, ValueRange(3, 10), ctype("char")
     ]
 
-* Duplicated annotations are not removed::
+* Duplicated metadata elements are not removed::
 
     Annotated[int, ValueRange(3, 10)] != Annotated[
         int, ValueRange(3, 10), ValueRange(3, 10)
     ]
 
-* ``Annotated`` can be used with nested and generic aliases::
+* ``Annotated`` can be used with nested and generic aliases, but only if it
```

Comment:
Perhaps this should clarify "...in the definition of nested and generic type aliases...". 

After all, it can be use "with aliases" (i.e. type aliases can be used within the base expression) even if the base expression is not a type expression. For example:

```python
class Foo[T](TypedDict):
    x: Annotated[Required[MyTypeAlias[T]], 1]
```